katas/content/single_qubit_gates/global_phase_i/Verification.qs

```diff
@@ -9,21 +9,21 @@ namespace Kata.Verification {
     }
 
     operation CheckSolution() : Bool {
-        let isCorrect = VerifySingleQubitOperation(Kata.GlobalPhaseI, GlobalPhaseI);
-
-        // Output different feedback to the user depending on whether the exercise was correct.
-        use target = Qubit[1];
-        let op = register => Kata.GlobalPhaseI(register[0]);
+        let solution = register => Kata.GlobalPhaseI(register[0]);
         let reference = register => GlobalPhaseI(register[0]);
+        let isCorrect = CheckOperationsEquivalenceStrict(solution, reference, 1);
+
+        // Output different feedback to the user depending on whether the solution was correct.
+        use target = Qubit[1]; // |0〉
```

Comment:
Same here, `target` and the corresponding `ResetAll` call can be moved into the else-block.